docs/guides/audio-config.md

```diff
@@ -2,13 +2,16 @@
 
 This page explains how to get the config files for using the T2 audio device, which allows use of the 3.5mm headphone port, the built in speakers, the built in mic and the headphones' mic.
 
+Before you proceed, make sure you already have `apple_bce` loaded by running `lsmod | grep apple_bce`. If not, follow the instructions on [how to setup the BCE module](https://wiki.t2linux.org/guides/audio-config).
```

Comment:
should this link be to the dkms guide instead? https://wiki.t2linux.org/guides/dkms/